scripts/soft-reboot

```diff
@@ -147,9 +147,17 @@ function setup_reboot_variables()
     fi
 }
 
+function invoke_kexec() {
+    /sbin/kexec -l "$KERNEL_IMAGE" --initrd="$INITRD" --append="$BOOT_OPTIONS" $@
+}
+
 function load_kernel() {
     # Load kernel into the memory
-    /sbin/kexec -l "$KERNEL_IMAGE" --initrd="$INITRD" --append="$BOOT_OPTIONS"
+    invoke_kexec -a
```

Comment:
>-a [](http://example.com/codeflow?start=17&length=2)

why changing? If this is valid and general bug fix, we need to evaluate the backport needs. Could you separate it into a standalone PR?

Reply:
Similar to above comment.